src/inmanta/server/services/compilerservice.py

```diff
@@ -728,6 +729,8 @@ async def request_recompile(
             they contain resources that are being exported.
         :param mergeable_env_vars: a set of env vars that can be compacted over multiple compiles.
             If multiple values are compacted, they will be joined using spaces.
+        :param links: An object that contains relevant links to this compile.
```

Comment:
I think ? If I understand correctly, these are links to services instances from this compile, right ?
```suggestion
        :param links: An object that contains relevant links for this compile.
```

Reply:
Yes, I believe that is correct